docs/guides/input.md

```diff
@@ -0,0 +1,294 @@
+# Input <Badge text="4.25+" />
+
+[:rocket: Quick Reference](#quick-reference)
+
+When you're building software, it's common to split things up into small pieces. These might be functions in a node process, or components in a React app. These small pieces often need to declare **dependencies** - bits of information/code that they're not responsible for, but that they rely on to serve their purpose.
```

Comment:
> things

What do you mean by “things” here? Is it “code”? “the project structure”? Something else? You give some examples in the next line, could we summarise the “things” with something more specific?

Reply:
```suggestion
When you're building software, it's common to split your code up into small pieces. These might be functions in a node process, or components in a React app. These small pieces often need to declare **dependencies** - bits of information/code that they're not responsible for, but that they rely on to serve their purpose.
```